profiles/export.cfg

```
@@ -0,0 +1,27 @@
+[buildout]
+extends = prod.cfg
+extensions += mr.developer
+auto-checkout +=
+    collective.exportimport
+
+[instance]
+eggs += 
+    collective.exportimport
+
+[sources]
+collective.exportimport = git https://github.com/collective/collective.exportimport.git
+
+[versions]
+attrs = 21.4.0
+backports.functools-lru-cache = 1.6.6
+beautifulsoup4 = 4.9.3
+functools32 = 3.2.3.post2
+hurry.filesize = 0.9
+ijson = 3.2.3
+jsonschema = 3.2.0
+plone.rest = 3.0.0
```

```suggestion
plone.api = 1.10.4
plone.rest = 3.0.0
```

collective.exportimport requires plone.api > 1.8.4